### DISCUSSION TITLE
Add support for tunes in each PSID file

### DISCUSSION THREAD

**Discussion by beachviking** [pschatzmann/SIDPlayer#9] [Ideas]
Each PSID data file has support for multiple tunes.  In the PSID header that is created during the load method, a default tune and the total number of tunes in the PSID file are specified.

This information can be stored in the SIDMetadata structure perhaps?

I am thinking that the default tune should be played as default and then the user should be able to use |next tune| or |previous tune|, in addition to the |next psid| and |previous psid| functionalities in a player.

**Comment by pschatzmann** [comment]
I extended the Stream class to support the subtune properly: 

- The requested subtune is part of the config. 
- The actual subtune is part of the Metadata.

Playing around with the subtune paramter however, I noticed  that this is only producing a short sound effect followed by silence.

So I am not sure if it is worth to extend the Player as well. 
But feel free to take this up...

**Comment by beachviking** [comment]
Ok, I will take a stab at proposing something in my fork of SIDPlayer.

**Reply by pschatzmann** [reply]
In this case you might also rely on the new isActive(timout) method to detect if audio is still produced...
